aggregator/aggregator.go

```diff
@@ -404,6 +444,29 @@ func (agg *aggregator) ownedShards() []*aggregatorShard {
 	return ownedShards
 }
 
+func (agg *aggregator) placement() (services.Placement, error) {
+	stagedPlacement, onStagedPlacementDoneFn, err := agg.placementWatcher.ActiveStagedPlacement()
```

Comment:
Both `ActiveStagedPlacement` and `ActivePlacement` calls are thread-safe so no locks are needed here.